packages/artillery-engine-playwright/index.js

```diff
@@ -352,6 +352,14 @@ class PlaywrightEngine {
           self.processor[spec.testFunction] ||
           self.processor[spec.flowFunction];
 
+        console.error('Playwright test function not found:', fn);
```

Comment:
@hassy I might be wrong, but why is that `console.error` call not inside the if block below ?

Reply:
because since I updated to v2.0.13 I get spammed a **lot** about some `test function not found` :smile:

Reply:
🤦‍♂️ and this is why even "trivial" changes should have an automated test. Will release a fix asap

Reply:
https://github.com/artilleryio/artillery/commit/2af901ea239a04bc42f309c4248a8ffda394aa5c

Reply:
no problem, it didn't break anything, just printed a lot :smile: